RDBM/Table.cpp

```diff
@@ -0,0 +1,225 @@
+#include"Table.h"
+bool strloss = false;
+Table::Table()
+{
+	id = ++TID;
+	name = "table " + std::to_string(id);
+}
+Table& Table::Create(std::vector<std::string> names, std::vector<char> coltypes)
```

Comment:
Bad practice to pass vector by copy. If it will be huge, it will take a lot of time to copy it.